aspnetcore-webapi/RateLimitingDotNET8/RateLimitingDotNET8/Program.cs

```diff
@@ -0,0 +1,42 @@
+using RateLimitingDotNET8;
+using System.Net;
```

Comment:
Remove unused using. Remove if it is present in any other files in the application